style_editor.html

```diff
@@ -8,14 +8,167 @@
   <title>ProffieOS Style Editor</title>
 <style>
 
-.button_on {
-  background: lightblue;
+/* Make page elements non-selectable with drag box */
+/* Took a vote, enabled won, but disabled for now. */
+/** {
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}*/
+
+/* Box sizing helps keep buttons uniform in
+size when padding and margins may change */
+* {
+  box-sizing: border-box;
+}
+
+/* Submit, copy, etc... */
+.middle-btn-container {
+  white-space: nowrap;
+}
+/* Expand and Layerize buttons */
+.middle-btn-container .button-off,
+.middle-btn-container .button-off:hover {
+  background-color: lightgray;
+  color: darkgray;
+}
+.middle-btn-container .button-on {
+  padding: 2px;
+  border: 2px solid dodgerblue;
+}
+/* Other buttons */
+button,
+input[type="button"] {
+  background-color: #ccc;
+  color: black;
+  border-radius: 2px;
+  padding: 3px;
+  font-size: 14px;
+  border: 1px solid #bbb;
+  font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
+  font-weight: normal;
+}
+button:hover,
+select:hover,
+input[type="button"]:hover {
+  background-color: #999;
+}
+button:active,
+input[type="button"]:active {
+  background-color: dodgerblue;
+}
+/* Power and Rotate latching buttons */
+.page-left .button-latched,
+.page-left .button-latched:hover {
+  background-color: dodgerblue;
+}
+/* Lockup and More Effects Drop Downs */
+select {
+  background-color: #ccc;
   color: black;
+  border-radius: 5px;
+  padding: 2px 0px;
+  font-size: 14px;
 }
 
-.button_off {
-  background: lightgray;
+/* Prevent Safari default style takeover at default zoom level */
+@media not all and (min-resolution:.001dpcm) { @media {
```

Comment:
media? Whut?

Reply:
Literally a StackOverflow find. It's supposed to keep the drop down menu buttons from being too small due to browser override at 100% zoom level. Any zoom change and they snap to adhere to the CSS rules. I *think* it seems to work, but truth be told, safari might need some massaging. Particularly iOS and the settings button later.

Reply:
I'm highly suspicious of stuff like this.
Do we need it?
Does it solve a real problem?
What happens without it?

Reply:
I think I subsequently covered it with CSS since adding that. Just tested. no issue at default zoom level. Removing.